app/helpers/helpers.rb

```diff
@@ -0,0 +1,15 @@
+class Helpers
+
+  def self.current_user(session_info)
```

Comment:
what we want is for these to be instance methods without arguments.

an example would be 

```ruby 
def current_user
    User.find(session[:id])
end
```  
and 
``` ruby 
def logged_in?
  !!session[:user_id]
end 
```